From: Yuanchu Xie <yuanchu@google.com>
Subject: selftests/damon: add damon to selftests root Makefile

Currently the damon selftests are not built with the rest of the
selftests.  We add damon to the list of targets.

Link: https://lkml.kernel.org/r/20220418202017.3583638-1-yuanchu@google.com
Fixes: b348eb7abd09 ("mm/damon: add user space selftests")
Signed-off-by: Yuanchu Xie <yuanchu@google.com>
Reviewed-by: SeongJae Park <sj@kernel.org>
Acked-by: David Rientjes <rientjes@google.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 tools/testing/selftests/Makefile |    1 +
 1 file changed, 1 insertion(+)

--- a/tools/testing/selftests/Makefile~selftests-damon-add-damon-to-selftests-root-makefile
+++ a/tools/testing/selftests/Makefile
@@ -9,6 +9,7 @@ TARGETS += clone3
 TARGETS += core
 TARGETS += cpufreq
 TARGETS += cpu-hotplug
+TARGETS += damon
 TARGETS += drivers/dma-buf
 TARGETS += efivarfs
 TARGETS += exec
_
